CHANGELOG.md

```diff
@@ -23,9 +23,14 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ### Added
 
-- [.netrc](https://ec.haxx.se/usingcurl/usingcurl-netrc) support. Calling `py42.sdk.from_local_account()` with no username and password will now result in py42 attempting to authenticate via an entry in your `.netrc` file, if you have one.
+- Methods for obtaining archive information:
+    - `sdk.archive.get_by_archive_guid`
```

Comment:
Some of these should have the workd `all` in there, like `get_all_by_device_guid`

Reply:
get_all_by_user_uid

Reply:
get_all_by_dest..